components/Feedback.js

```diff
@@ -71,7 +71,7 @@ export default function Feedback(className = '') {
             <div className='block mx-auto w-fit'>
                 <img src='/img/illustrations/icons/icon-x.svg' className='md:w-14' />
             </div>
-            <div className='text-center mx-auto text-lg mt-4'>
+            <div className='text-center mx-auto text-lg mt-4' data-testid="Feedback-div2">
```

Comment:
Making naming convention with 1,2,3.. as suffux is not good practice, kindly update it with good naming for `data-testid`.